docker-test-framework/etc/irods-ext/metalnx.properties

```diff
@@ -175,4 +175,22 @@ pluggablepublishing.info.timeout=0
 pluggablepublishing.publishing.timeout=0
 
 # server rule engine instance that will provide the galleryview listing
-gallery_view.rule_engine_plugin.instance_name=irods_rule_engine_plugin-irods_rule_language-instance
\ No newline at end of file
+gallery_view.rule_engine_plugin.instance_name=irods_rule_engine_plugin-irods_rule_language-instance
+
+##########################################################
+# Metadata Masking Properties
+#
+# Excludes metadata when the attribute name starts with at least one prefix.
+# Multiple prefixes can be defined by separating them with the character sequence defined
+# by metalnx.metadata.mask.delimiter.
+#
+# For example, the configuration below will hide any metadata that contains an attribute
+# starting with "irods::", "metalnx-", or "_system_".
+#
+#     metalnx.metadata.mask.prefixes=irods::||metalnx-||_system_
+#     metalnx.metadata.mask.delimiter=||
+#
+# Use the iRODS Metadata Guard rule engine plugin to protect your metadata namespaces from
+# being modified.
+metalnx.metadata.mask.prefixes=irods::
+metalnx.metadata.mask.delimiter=;
```

Comment:
perhaps a comma is more expected as a default delimiter?

Reply:
I don't have any strong opinions about it.

Do you prefer a comma?

Reply:
done